.github/workflows/ephemeral-env-cleanup.yml

```diff
@@ -14,6 +14,9 @@
 
 name: Ephemeral environment - cleanup
```

Comment:
Region should be configurable in vars, if someone is going to create a fork, may be interested in other region dlan europe

Reply:
Fixed :)

Reply:
Docs added, extracted variables connected with GCP account.